src/templates/project-details.js

```diff
@@ -1,83 +1,28 @@
 import React from 'react';
 import PropTypes from 'prop-types';
-import classnames from 'classnames';
 import Layout from '../components/Layout';
-import Image from '../components/LazyImage'
-import ProjectDetailButtonChild from '../components/project/DetailButtonChild';
-import TwitterButton from '../components/project/TwitterButton';
-import s from './styles.module.scss';
+import ProjectOverview from './project-overview';
```

Comment:
File name is usually named just like the component. `ProjectOverview.js` instead of `project-overview.js`

Reply:
Ok, I was following the existing `project-details.js`, I am renaming this as well than.